habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -960,11 +960,11 @@ class SimulatorConfig(HabitatBaseConfig):
     # Rearrange agent grasping
     hold_thresh: float = 0.09
     grasp_impulse: float = 1000.0
-    agents: List[str] = field(default_factory=lambda: ["agent_0"])
-    agent_0: AgentConfig = AgentConfig()
-    agent_1: AgentConfig = (
-        MISSING  # temporary multi-agent rearrangement config fix
-    )
+    # we assume agent(s) to be set explicitly
+    agents: Dict[str, AgentConfig] = MISSING
```

Comment:
```suggestion
    agents: List[AgentConfig] = [MISSING]
```
Why not that? Then agent order is defined, each agent has a unique ID given by its order in the list, right? Or is there some reason we can't do that. In the event taht does not work, does Hydrasupport that Dict just being an OrderedDict instead? ANd then the order still matters?